scripts/make_slack_notification_message.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env python
+"""Format summary files for Slack perusal. Only uses stdlib."""
```

Comment:
Might be nice to have a tiny bit of documentation about the outputs from this script. Doesn't need to be much, but a quick overview of how the summaries are formatted could be helpful if someone comes in to change that in the future.